# PCR primer design for Fusion PCR, Gibson assembly or Homologous recombination

In this exercise, we will learn how to design PCR primers in order to join two DNA fragments together by three methods: Fusion PCR, Gibson assembly and homologous recombination. These  methods work by differently, but the basic design principles is the same.

## **Fusion PCR**
Fusion or overlap extension PCR is perhaps the fastest way to join two DNA molecules together. Only a DNA polymerase and dNTPs are needed together with specially designed PCR primers, see below.
![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_001.png]]

With more detials, two PCR products are made in separate reactions (blue and purple boxes below). These are mixed in a third PCR reaction (green box) where the two fragments join in the middle and amplify without PCR primers.


![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_002.png]]

## **Gibson assembly**

Gibson assembly is a modern cloning method to join two or more DNA fragments together. This method  was first described in a [publication](https://www.nature.com/articles/nmeth.1318) from 2009. The method does not require restriction enzymes, but relies on a engineering the DNA fragments with PCR and adding a mixture three enzymes:

- Exonuclease
- DNA polymerase
- DNA ligase.

This mixture is sometimes called a *Gibson assembly mastermix*. These mixtures are commercially available from many companies.

![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_003.png]]

The DNA molecules have to have overlapping flanking sequences to work. The green and black pieces of DNA are engineered using PCR in order to have overlapping flanking sequences of about 40 bp. **This is the exact same procedure as for fusion PCR**.

All DNA fragments are mixed in a test tube together with Gibson assembly master mix. The DNA fragments are first partially degraded by a 5‚Äô-3‚Äô exonuclease. After the molecules anneal, a DNA polymerase fills in the gaps and a DNA ligase seals the phosphodiester bonds. A video is available on YouTube (2 min) that explains the process in detail (below).


![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_004.png]]

[video link](https://www.youtube.com/watch?v=tlVbf5fXhp4)

## Homologous recombination

Homologous recombination is essentially Gibson assembly *in-vivo*. The baker‚Äôs yeast *Saccharomyces cerevisiae* is very good at doing this. Yeast homologous recombination is considered to be the most robust way to join several DNA molecules. Below, three linear DNA fragments are joined together by homologous recombination.  A drawback of the method is that yeast cells grow slower than *E. coli.*


![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_006.png]]




The DNA repair gene RAD52 encodes a key protein for this process (below). The process is essentially the same as for Gibson assembly, but happens inside the cell instead of in a test tube.

![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_007.png]]



> [!IMPORTANT]
> don‚Äôt skip the example below.

## Primer design

The primer design process is outlined in detail below. In this example We would like to join the red and the blue pieces of DNA. To do this we need four primers:

- red_forward
- red_reverse
- blue_forward
- blue_reverse

![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_008.png]]

The annealing part of each primer should be six nucleotides long. This means that the length of red_forward and blue_reverse primers are six nucleotides. The length of the overlapping tails should be five nucleotides. This means that red_reverse and blue forward primers should be 6 + 5 = 11 nucleotides long. We are free to chose any primer length and tail length we want.

The sequences of the red (upper left) and blue (lower right) part are these :

```
             ctgtgccagatcgtctggca

                                 tctcgaaactatagtcgtacagatcgaaa
```

We can design primers for this experiment quite easily with a simple trick. Copy the sequence above into the same ApE window as below. The example has features added that color each sequence, this only for clarity and is not strictly necessary:

![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_009.png]]

Select the "blue_forward‚Äù primer by selecting five nucleotides of the red sequence and (tail) and six nucleotides of the blue:

![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_010.png]]

The copied text should be `tggcatctcga`.

Now select six nucleotides of the red and five of the blue:

![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_012.png]]

Now select Edit>Copy Rev-Com from the ApE task bar:

![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_013.png]]

The copied sequence should be the one below:

![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_001-1.png]]

The ‚Äúred_forward‚Äù is selected from the red sequence without modifications:

![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_002-1.png]]

![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_003-1.png]]

and ‚Äúblue_reverse‚Äù primers is copied using Edit>Copy Rev-Com:

![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_004-1.png]]

![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_005-1.png]]

The four resulting primers together:

![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_006-1.png]]

There is a video on YouTube (12 min) showing the same procedure (below).

![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_007-1.png]]
[video link](https://youtu.be/eLJLKyHFqTg?t=39)

**Question 1:** Design four primers (a, b, c and d) for the Gibson assembly of the following two double stranded DNA molecules (x and y) joined together in the order shown:

```
agtcagtcgtagtctagtcgtgtaag

                          ctctctattcaatccatcatcta
```

![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_008-1.png]]

The annealing part of each primer should be 6 nucleotides long. That are the same sizes as the primers in the previous example. This means that The length of primer a and d should be 6 nucleotides. The overlapping part formed by the tails should be seven nucleotides long . This means that primers b and c should be 6+7 = 13 nucleotides long.

Fill in the missing information in the table below:

| Sequence | Length | Partial seguid    | Complete seguid |
| :------- | :----- | :---------------- | :-------------- |
| a        | 6      | `lsseguid=gd7zyw` | ?               |
| b        | 13     | `lsseguid=eWpW6o` | ?               |
| c        | 13     | `lsseguid=m6LOMc` | ?               |
| d        | 6      | `lsseguid=waQ2DA` | ?               |


**Question 2:**

This is an individual question for each student. Got to the Google Spreadsheet for this question. You should find your name in the leftmost column. The columns called **sequence_a, sequence_b** which contain two DNA sequences that should be joined by Gibson assembly just like in the previous example. Your task is to design the **four PCR primers** required to do this.

![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_010-1.png]]

The length of primer fw_a and rv_b should be 6 nucleotides. The overlapping part formed by the tails should be six nucleotides long. This means that primers b and c should be 6+6 = 12 nucleotides long.




üíÉüèªüï∫üèΩ

![[TPs/TP09_Cloning_by_HR_and_Gibson_assembly/TP09_005.png]]

[video link](https://youtu.be/WCWjJFU1be8)



¬© Bj√∂rn Johansson 2013-2025