## ***Primer design and melting temperature calculations for PCR primers***
An exemplary PCR program is shown below. It is important to adjust the annealing temperature (50ºC in the example) according to the primers used. Primers have a certain melting temperature (Tm) which depend mostly on the GC content and length of the primer. This melting temperature will affect the annealing temperature chosen for the PCR reaction.



![](Aspose.Words.4ff61986-4dab-4633-998d-cf3d99ed5ad3.001.png)






We need the Tm for both primers in order to calculate the annealing temperature (Ta) of the PCR reaction. The simplest formula that can be used is called the **Basic formula**:

Tm = (A+T) \* 2 + (G+C) \* 4

A, T, G and C  are the number of the respective bases in the sequence (modified from **Marmur**, J., and Doty, P. 1962 *J Mol Biol* **5**:109-118).  According to this formula, the Tm for the primer below should be **42ºC**:

**>primer**

**gatcatctcgatcta**

You can calculate this manually or use the simple Excel sheet called “BasicTm.xlsx” that can be found in the folder of this file.



**Question 1:**

What is the Tm of the primer2 below using the basic formula?

**>primer2**

**cacttcctgacatcg**

There are also more elaborate formulas for Tm calculation. These are often implemented in special software tools. There are many such tools online, but it is not always clear which formula is used and what the parameters are.

One open source and completely transparent tool is the Tm tool of WebPCR (Fig 2) that can be found online: <http://pydna.pythonanywhere.com/tm>.

The advantage of this tool is that it uses the open-source Biopython algorithms, specifically the [Bio.SeqUtils.MeltingTemp module](https://biopython.org/docs/1.75/api/Bio.SeqUtils.MeltingTemp.html#module-Bio.SeqUtils.MeltingTemp) which are freely available.

![](Aspose.Words.4ff61986-4dab-4633-998d-cf3d99ed5ad3.003.png)
![](Aspose.Words.4ff61986-4dab-4633-998d-cf3d99ed5ad3.002.png)








**Question 2:**

What is the Tm of the oligonucleotide <a name="docs-internal-guid-ea9ef415-7fff-6a3b-389c-9b75d77a2ab2"></a>**gttctgatcctcgagcatcttaagaattc** using the Tm tool of WebPCR and the settings indicated in (Fig 2).


**Question 3:**

This is an individual question for each student. Follow this [link](https://docs.google.com/spreadsheets/d/1ximPJSlAnsjHp_KAFh2-kr2Nz-VnfHfO_AjdXtUANQM/edit#gid=0) that points to a Google Spreadsheet.   You should find your name in the leftmost column.

One column called **geneY** contains a DNA sequence that represents a double stranded **linear** DNA molecule that is also an open reading frame. Your task is to design two primers (**fp** and **rp**) for geneY** that will amplify the sequence.

The primers should have a melting temperature between 48 and 52C using the basic formula.

Put your results in the indicated cells for forward primer (**fp**), reverse primer (**rp**).

Simulate the cloning of the gene using the plasmid vector pYPKa. Use the restriction enzyme **AjiI** which is the same as **BtrI** or **BmgBI**. The pYPKa sequence can be found in the file “pYPKa.gb” which can be found in the same folder as this file.

Please answer with raw DNA sequences as indicated for the first example student "*Max Maximus*".


