The accepted genetic nomenclature for chromosomal genes of the yeast *S. cerevisiae* is illustrated in the table above, using *ARG2* as an example. Entries in the table marked with ✨ were added by us.

|     | Gene symbol                           | Definition                                                                                                       |
| --- | ------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
|     | *ARG2*                                | **Locus or dominant allele**                                                                                     |
|     | *arg2*                                | **Locus or recessive allele conferring an arginine requirement**                                                 |
|     | *arg2-∆1*                             | Specific complete or partial deletion of ARG2                                                                    |
|     | *arg2::Kan*MX4                        | **Insertion of the functional KanMX4 gene at the ARG2 locus; arg2 is or became nonfunctional**                   |
|     | *arg2::LEU2*                          | **Insertion of the functional LEU2 gene at the ARG2 locus; arg2 is or became nonfunctional**                     |
|     | P<sub>CYC1</sub>-*ARG2*               | Fusion between the CYC1 **promoter** and ARG2, where the ARG2 gene is functional                                 |
| ✨   | JEN1-GFP                              | Fusion between the JEN1 coding gene and the GFP coding gene expressed as a fusion protein (one polypeptide).     |
|     | *ARG<sup>+</sup>*                     | All wild-type alleles controlling arginine requirement                                                           |
|     | *arg2<sup>-</sup>*                    | Any arg2 allele conferring an arginine requirement                                                               |
|     | *ARG2<sup>+</sup>*                    | Wild-type allele                                                                                                 |
|     | *arg2-9*                              | Specific allele or mutation                                                                                      |
|     | Arg<sup>+</sup>                       | Strain not requiring arginine                                                                                    |
|     | Arg<sup>-</sup>                       | Strain requiring arginine                                                                                        |
|     | Arg2p                                 | Protein encoded by ARG2                                                                                          |
|     | Arg2 protein                          | Protein encoded by ARG2                                                                                          |
|     | *ARG2* mRNA                           | mRNA transcribed from ARG2                                                                                       |
|     | *ARG2::LEU2*                          | Insertion of the functional LEU2 gene at the ARG2 locus; ARG2 remains functional and dominant                    |
|     | *arg2-10::LEU2*                       | Insertion of the functional LEU2 gene at the ARG2 locus, and the specified arg2-10 allele which is nonfunctional |
|     | *cyc1-arg2*                           | Fusion between the CYC1 and ARG2 **genes**, where both are nonfunctional                                         |
|     | MATa/MATα FAS2 / fas2::KanMX4         | Heterozygous deletion in Diploid strain                                                                          |
|     | MATa/MATα fas2::KanMX4 / fas2::NatMX4 | Homozygous deletion in Diploid strain                                                                            |

From [Sherman, Fred. 2002. “Getting Started with Yeast.” *Methods in Enzymology* 350: 3–41.](https://www.google.com/search?as_q=Sherman%2C+Fred.+2002.+%E2%80%9CGetting+Started+with+Yeast.%E2%80%9D+*Methods+in+Enzymology*+350%3A+3%E2%80%9341.&as_epq=&as_oq=&as_eq=&as_nlo=&as_nhi=&lr=&cr=&as_qdr=all&as_sitesearch=&as_occt=any&safe=images&as_filetype=pdf&tbs=)

Whenever possible, each gene, allele, or locus is designated by three *italicized* letters, e.g., *ARG*, which is usually a describer of the function, followed by a number, e.g., ARG2.

Unlike most other systems of genetic nomenclature, dominant alleles are denoted by using UPPERCASE *italics* for all letters of the gene symbol, e.g., *ARG2*, whereas lowercase letters denote the **recessive** allele, e.g., the auxotrophic marker *arg2*. Wild-type genes are designated with a superscript "plus" (*sup6<sup>+</sup>* or *ARG2<sup>+</sup>*).

[**Alleles**](https://en.wikipedia.org/wiki/Allele) are designated by a number separated from the locus number by a hyphen (-), e.g., arg2-9.

The symbol ∆ (greek capital letter delta) can denote complete or partial deletions, e.g., arg2-∆1. (Do **not** use the symbols ∆arg2 or arg2∆ for deletions.)

Insertion of genes follow the bacterial nomenclature by using the symbol **::**. For example, *arg2*::*LEU2* denotes the insertion of the *LEU2* gene at the *ARG2* locus in which *LEU2* is **dominant** (and **functional**) and *arg2* is **recessive** (and **defective**).

Phenotypes are denoted by cognate symbols in roman type and by the superscripts **+** and **-**. For example, the independence and requirement for arginine can be denoted by Arg<sup>+</sup> and Arg<sup>-</sup>, respectively.

Proteins encoded by ARG2, for example, can be denoted Arg2p, or simply Arg2 protein. However, gene symbols are generally used as adjectives for other nouns, for example, ARG2 mRNA or  ARG2 strains. Resistance and sensitivity phenotypes are designated by superscript R and S, respectively. For example, resistance and sensitivity to canavanine sulfate are designated Can<sup>R</sup> and Can<sup>S</sup>, respectively.

Although most alleles can be unambiguously assigned as dominant or recessive by examining the phenotype of the heterozygous diploid crosses, dominant and recessive traits are defined only with pairs, and a single allele can be both dominant and recessive. For example, because the alleles *CYC1+* , *cyc1-717*, and *cyc1-∆1* produce, respectively, 100%, 5 %, and 0% of the gene product, the *cyc1-717* allele can be considered recessive in the *cyc1-717*/*CYC1+* cross and dominant in the *CYC1-717*/*cyc1-∆1* cross. Thus, it is less confusing to denote all mutant alleles by lowercase letters, especially when considering a series of mutations having a range of activities.

Auxiliary gene symbols can be used to further describe the corresponding phenotypes, including the use of superscript R and S to distinguish genes conferring resistance and sensitivity, respectively. For example, the genes controlling resistance to canavanine sulfate (*can1*) and copper sulfate (*CUP1*) and their sensitive alleles could be denoted, respectively, as *can<sup>R</sup>1*, *CUP<sup>R</sup>1*, *CAN<sup>S</sup>1*, and *cup<sup>S</sup>1*.
