# TP18 Reproducing a cloning strategy from literature

Scientific journal articles should provide sufficient information for the reproduction of cloning and vector construction experiments. One of the foundational principles of science is that experiments should be reproducible. If other researchers cannot replicate the results from a published paper, then the findings cannot be reliably verified or built upon.

Reproducibility ensures that the methods described in the paper are accurate and that the results are not due to error or bias. If cloning protocols or vector construction details are omitted or unclear, it opens the door to misinterpretations or mistakes in following the methods. For example, the wrong plasmid backbone, cloning site, or even incorrect conditions could lead to failures in replication.

This exercise contain three examples of journal articles that report the study of a plasmid vector. Your task is to try to replicate the cloning steps *in-silico*.
# Assembly of the YEp24PGK_XK vector

Out first task is to reproduce the construction of a plasmid called YEp24PGK_XK constructed as a part of the work reported in the article below:

![[TPs/TP18_Reproducing_a_cloning_strategy_from_literature/TP18_001-1.png]]

The complete reference is:

Johansson, B, C Christensson, T Hobley, and B Hahn-Hägerdal. 2001. **Xylulokinase Overexpression in Two Strains of Saccharomyces Cerevisiae Also Expressing Xylose Reductase and Xylitol Dehydrogenase and Its Effect on Fermentation of Xylose and Lignocellulosic Hydrolysate.** Appl. Environ. Microbiol. 67 (9): 4249–55.

The construction of the vector is described in the MATERIALS AND METHODS section of the article. Briefly, the XKS1 gene from _Saccharomyces cerevisiae_ was amplified by PCR using two primers called **primer1** and **primer3**. The primers add restriction sites for BamHI to the ends of the XKS1 gene.

The PCR product was digested with BamHI and ligated to the YEp24PGK plasmid that has previously been digested with BglII which cut the plasmid in one location. See the figure below for more details:

![[TPs/TP18_Reproducing_a_cloning_strategy_from_literature/TP18_002.png]]

### Question 1
Use the information in the article to assemble the YEp24PGK_XK vector sequence. Put size and checksum for the resulting vector in the Google sheet. Sequences can be obtained from SGD (Saccharomyces Genome Database) or Genbank.

### Motivation

Studying D-xylose metabolism in recombinant *Saccharomyces cerevisiae* (baker's yeast) is important for several key reasons, especially in the context of metabolic engineering, industrial biotechnology, and synthetic biology. D-xylose is a five-carbon sugar that is a key component of lignocellulosic biomass. Understanding how *S. cerevisiae* can metabolize D-xulose is critical for developing strains of yeast capable of efficiently converting these sugars into biofuels such as ethanol. Since natural *S. cerevisiae* strains don't have a native pathway to metabolize D-xylose, engineering recombinant strains to process these sugars can broaden the range of feedstocks that can be used for biofuel production.

# Assembly of the pGUP1 vector

The construction of the vector pGUP1 is described in the publication below:

![[TPs/TP18_Reproducing_a_cloning_strategy_from_literature/TP18_003.png]]

The complete reference is:

Régine Bosson, Malika Jaquenoud, and Andreas Conzelmann, **GUP1 of Saccharomyces cerevisiae Encodes an O-acyltransferase Involved in Remodeling of the GPI Anchor**, Molecular Biology of the Cell 17, no. 6 (June 2006): 2636–2645.

The cloning is described in the paper on the second page of the article (page 2637) on the upper left side of the page.

Briefly, the GUP1 gene from *S. cerevisiae* was amplified with two primers GUP1rec1sens and GUP1rec2AS and recombined with the pGREG505 vector. The pGREG505 was first digested with *Sal*I and only the big fragment was used for the recombination.

![[TPs/TP18_Reproducing_a_cloning_strategy_from_literature/TP18_004.png]]



> [!Tip!]
> The article contain a link for the pGREG505 sequence in the end of the Materials and methods
> section that is now inactive. Use this [link](http://www.euroscarf.de/plasmid_details.php?accno=P30350) instead.

### Question 2
Use the information in the article to assemble the pGUP1 plasid sequence. Put size and checksum for the resulting vector in the Google sheet. Sequences can be obtained from SGD (Saccharomyces Genome Database) or Genbank.

### Motivation
The GUP1 gene in Saccharomyces cerevisiae (baker's yeast) encodes a protein that is involved in the biosynthesis of GPI (glycosylphosphatidylinositol) anchors. These anchors are important for attaching certain proteins to the outer membrane of the cell.

![[TPs/TP18_Reproducing_a_cloning_strategy_from_literature/TP18_007.png]]

# Assembly of the p416TEF vector

The "Mumberg vectors" are a popular vector collection for gene expression in *S. cerevisiae*. Their construction was described in the article below:


![[TPs/TP18_Reproducing_a_cloning_strategy_from_literature/TP18_005.png]]


The p416TEF vector has a TEF2 promoter and a CYC1 terminator controlling the expression of genes that can be cloned using a number of different restriction sites (MCS).


![[TPs/TP18_Reproducing_a_cloning_strategy_from_literature/TP18_001.png]]

The p416TEF was constructed from a vector called pRS416.


**Question3**:

Follow the cloning strategy described in Mumberg 1995 to assemble the sequence of the plasmid p416TEF. Two linear DNA fragments are given below that are useful.

```

>CYC1 terminator XhoI KpnI fragmen
CTCGAGTCATGTAATTAGTTATGTCACGCTTACATTCACGCCCTCCCCCCACATCCGCTCTAACCGAAAAGGAAGGAGTTAGACAACCTG
AAGTCTAGGTCCCTATTTATTTTTTTATAGTTATGTTAGTATTAAGAACGTTATTTATATTTCAAATTTTTCTTTTTTTTCTGTACAGAC
GCGTGTACGCATGTAACATTATACTGAAAACCTTGCTTGAGAAGGTTTTGGGACGCTCGAAGGCTTTAATTTGCGGCCGGTACC


>TEF2 promoter PCR product SacI XbaI fragmen
GAGCTCTTACCCATAAGGTTGTTTGTGACGGCGTCGTACAAGAGAACGTGGGAACTTTTTAGGCTCACCAAAAAAGAAAGAAAAAATACG
AGTTGCTGACAGAAGCCTCAAGAAAAAAAAAATTCTTCTTCGACTATGCTGGAGGCAGAGATGATCGAGCCGGTAGTTAACTATATATAG
CTAAATTGGTTCCATCACCTTCTTTTCTGGTGTCGCTCCTTCTAGTGCTATTTCTGGCTTTTCCTATTTTTTTTTTTCCATTTTTCTTTC
TCTCTTTCTAATATATAAATTCTCTTGCATTTTCTATTTTTCTCTCTATCTATTCTACTTGTTTATTCCCTTCAAGGTTTTTTTTTAAGG
AGTACTTGTTTTTAGAATATACGGTCAACGAACTATAATTAACTAAACTCTAGA

```
## Further reading

![[TPs/TP18_Reproducing_a_cloning_strategy_from_literature/TP18_006.png]]

[Why Is So Much Cloning Documentation Wrong?](https://www.anaconda.com/blog/why-is-so-much-cloning-documentation-wrong) is a blog post from 2022 where I describe the problem of incomplete documentation and
give an example of how to create complete, executable documentation with the help of [Pydna](https://github.com/pydna-group/pydna) and [Python](https://www.python.org/).

© Björn Johansson 2013-2025
