# PCR prediction given primer and template sequences

The objective of this exercise is to show **how to assemble the sequence of a PCR product when you have access to both sequences of the primers and the template**.

Primers, which are single stranded DNA molecules used in PCR, are usually at least 16 nucleotides (nt) long and template sequences are **usually** at least 100 nt long. In the examples in this document, both primers and template are **much shorter** for clarity.

It is necessary to have a basic understanding of the principles of PCR before solving the exercises.

## A simple example

The primers “forward” and “reverse” can be used to amplify the DNA molecule “template” below. The sequences are given in [[FASTA]] format.
```
>forward
acgtac

>reverse
tagcta

>template
acgtactactactagctagcgtatcatcgtagctagcta
```

The sequences for the primers represent single stranded DNA while the **template** is double stranded and represents this molecule in reality (below):

```
acgtactactactagctagcgtatcatcgtagctagcta Watson
|||||||||||||||||||||||||||||||||||||||
|||||||||||||||||||||||||||||||||||||||
tgcatgatgatgatcgatcgcatagtagcatcgatcgat Crick
```

We call the upper strand **Watson** and the lower strand **Crick** in the example above. In the first cycle of PCR, the strands are separated by heating. After cooling, the forward and reverse primers then anneal as shown below.

The “forward” primer anneals to the **Crick** strand while the “reverse” primer anneals to the **Watson** strand (below).

```
acgtactactactagctagcgtatcatcgtagctagcta Watson
|||||||||||||||||||||||||||||||||||||||
                                <atcga
acgtac>
|||||||||||||||||||||||||||||||||||||||
tgcatgatgatgatcgatcgcatagtagcatcgatcgat Crick
```

Compare the reverse primer in above with the with the one in the FASTA sequence. It is evident that the sequence above is the reverse of the one in  the FASTA sequence. DNA polymerase (such as Taq) synthesize in the 5'-3' direction (indicated by **<** and **>**).

![[TPs/TP04_Manual_PCR_simulation/TP04_001-1.png]]

<!--
```
acgtactactactagctagcgtatcatcgtagctagcta Watson
|||||||||||||||||||||||||||||||||||||||
|||||||||||||||||||||||||||||||||||||||
tgcatgatgatgatcgatcgcatagtagcatcgatcgat Crick (new)

acgtactactactagctagcgtatcatcgtagctagcta Watson (new)
|||||||||||||||||||||||||||||||||||||||
|||||||||||||||||||||||||||||||||||||||
tgcatgatgatgatcgatcgcatagtagcatcgatcgat Crick
```
-->
As you can see above, the primers are incorporated in the newly synthesized DNA strands and become a part of the final PCR product. Theoretically, in each cycle, one new DNA strand (Watson or Crick) is produced from each existing strand, so that there is a doubling of strands with each cycle. This doubling of the number of DNA molecules with each cycle is a exponential increase with time and the reason why the process is called “**chain reaction**”.

## Forward and Reverse primers

It is important to understand that what we call forward and reverse primer is **simply a convention**. As you probably know we can choose to view the crick strand instead of the Watson strand:

```
tagctagctacgatgatacgctagctagtagtagtacg
|||||||||||||||||||||||||||||||||||||||
|||||||||||||||||||||||||||||||||||||||
atcgatcgatgctactatgcgatcgatcatcatcatgca
```

This is exactly the same molecule as in  the previous example. The primers now anneal like this:

```
tagctagctacgatgatacgctagctagtagtagtacg
|||||||||||||||||||||||||||||||||||||||
                                <catgca
tagcta>
|||||||||||||||||||||||||||||||||||||||
atcgatcgatgctactatgcgatcgatcatcatcatgca
```

Note that the previous forward primer is now the reverse primer and that the previous reverse primer is now the forward primer.

**Question 1**:

Predict the sequence of the PCR product from the primers and template below. Note that the primers do not anneal at the end of the template, you have to find the annealing locations manually.

```
>f
aactatc

>r
ctatcg

>
cgaactatccacatctcgataga
```

What is the size of the PCR product? The partial seguid of the answer is `ldseguid=uYStiF`
What is the complete checksum?

## **A Very Important Tip That Will Save You Time!**

It is not necessary to produce an elaborate figure as the one above. Suppose you have the primers and the template below:

```
>forward primer
gatcagc
>reverse primer
ccactcata
>template
gggatcagccgaactatcgacatctcgaatatgagtggtaga
```

First find the forward primer in the template (indicated by red):

![[TPs/TP04_Manual_PCR_simulation/TP04_003.png]]

Then make the reverse-complement of the reverse primer:

![[TPs/TP04_Manual_PCR_simulation/TP04_002.png]]

Find the reverse-complement of the reverse primer in the template:

![[TPs/TP04_Manual_PCR_simulation/TP04_004.png]]


The sequence of the PCR product is the sequence between the colored regions above which is  between the primers including the primers.

> [!TIP]
>You can of do the reverse in order to *design* PCR primers for a certain sequence.


## ***Design primers to amplify a specific sequence***

**Question 2**:

Design two primers called F2 and R2 that are each **six** nucleotides long to amplify the entire template sequence:

```
>template
tgatctactgatcaatatgccgacgagac

>F2
?

>R2
?
```

| Sequence | Length | Partial seguid  | Complete seguid |
| :------- | :----- | :-------------- | :-------------- |
| F2       | 6      | lsseguid=sPQWcm | ?               |
| R2       | 6      | lsseguid=UwV1HG | ?               |
| Product3 | 30     | ldseguid=6we_Wk | ?               |

**Question 3**:

Design two primers called F3 and R3 that are each six nucleotides long to amplify the template sequence from (and including) nucleotide 6 to 25, so that the PCR product is exactly 20 nucleotides long.

Tip! You can use ApE for this, choose Edit>Select From-To... and enter 6 and 25. Assemble the sequence of the PCR product as well

```
>template
tgatctactgatcaatatgccgacgagac

>F3
?

>R3
?

>Product3
?
```

Fill in the missing information:

| Sequence | Length | Partial seguid  | Complete seguid |
| :------- | :----- | :-------------- | :-------------- |
| F3       | 6      | lsseguid=ep4f2i | ?               |
| R3       | 6      | lsseguid=I-CUjD | ?               |
| Product3 | 20     | ldseguid=C1mAEt | ?               |

## ***Design primers for engineering the ends of the PCR product***

PCR can be used to add specific sequences that are not part of the template to the ends of the PCR product.

In this example, we add restriction sites to the PCR product. This is a common method to clone a PCR product in a vector.

Consider the following **double** stranded DNA molecule:

```
>template
acgtactactactagctagcgtatcatcgtagctagcta
```

If we would like to clone the molecule above (which is same as in the first example of this document) we would like to have restriction enzyme sites at the ends of the molecule.

We design the PCR primers (below).
```
>forward
acgtac
>reverse
tagcta
```

We need an enzyme that does NOT cut in the interior of the DNA fragment. One such enzyme is *Bam*HI (you can check this using ApE). The recognition sequence of *Bam*HI is `GGATCC`.

We simply add this sequence to the primers in the 5'side:

```
>forward
GGATCCacgtac
>reverse
GGATCCtagcta
```

The new primers will anneal like this in the very first cycle of PCR:

```
acgtactactactagctagcgtatcatcgtagctagcta>
|||||||||||||||||||||||||||||||||||||||
                                 ||||||
                                <atcgatCCTAGG

GGATCCacgtac>
      ||||||
      |||||||||||||||||||||||||||||||||||||||
     <tgcatgatgatgatcgatcgcatagtagcatcgatcga
```
Note that the extra nucleotides we added do not anneal to the template. DNA polymerase will synthesize new DNA in four places **(*> < > <***). The final molecules will have these structures:

```
GGATCCacgtactactactagctagcgtatcatcgtagctagcta        (a)
|||||||||||||||||||||||||||||||||||||||||||||
|||||||||||||||||||||||||||||||||||||||||||||
CCTAGGtgcatgatgatgatcgatcgcatagtagcatcgatcga

      acgtactactactagctagcgtatcatcgtagctagctaGGATCC
      |||||||||||||||||||||||||||||||||||||||||||||
      |||||||||||||||||||||||||||||||||||||||||||||
      tgcatgatgatgatcgatcgcatagtagcatcgatcgatCCTAGG  (b)

```

We have two molecules, each with the restriction site in the end. In the next cycle, sequence **a** and **b** above will anneal and produce the final product.

```
GGATCCacgtactactactagctagcgtatcatcgtagctagcta>
|||||||||||||||||||||||||||||||||||||||||||||
      ||||||||||||||||||||||||||||||||||||||||||||
    <tgcatgatgatgatcgatcgcatagtagcatcgatcgatCCTAGG
```

The final product is shown below:

```
GGATCCacgtactactactagctagcgtatcatcgtagctagctaGGATCC
CCTAGGtgcatgatgatgatcgatcgcatagtagcatcgatcgatCCTAGG
```

or in FASTA format_

```
>template
GGATCCacgtactactactagctagcgtatcatcgtagctagctaGGATCC
```


**Question 4:**

Design primers for PCR engineering of *Eco*RI sites to the ends of the DNA sequence below (`template4`). The primers (F4 and R4) should be **six** nucleotides long not inluding the EcoRI site sequences.

You can find information about *Eco*RI in ApE, choose Enzymes>Enzyme Selector... Assemble the sequences of F4, R4 and Product4 and fill in the missing information in the table below:

```
>template4
atatgcatcatctatctacgtagcgtatgctataatcta

>F4
?

>R4
?

>Product4
?
```

| Sequence | Length | Partial seguid   | Complete seguid |
| :------- | :----- | :--------------- | :-------------- |
| F4       | 12     | lsseguid=xOCQ9A  | ?               |
| R4       | 12     | lsseguid=8Slf72j | ?               |
| Product4 | 51     | ldseguid=L3hF3m  | ?               |

**Question 5:**

The sequence `template5` is circular. Assemble the sequence of the PCR product (`Product5`) using primers F5 and R5. Fill in the missing information in the table below:

```
>template5 circular
atatgcatcatctatctacgtagcgtatgctataatcta

>F5
tagcg

>R5
agatga

>Product5
?
```

| Sequence | Length | Partial seguid  | Complete seguid |
| :------- | :----- | :-------------- | :-------------- |
| Product5 | 32     | lsseguid=xHaBSo | ?               |

**Question 6:**

This is an individual question for each student. You should find your name in the leftmost column in the Google sheet for TP04.

The goal of this exercise is to design two primers for a sequence called **geneX**. The primers should add flanking restriction sites for two restriction enzymes, **Enz#1** and **Enz#2.**

These sites are also present in the vector pUCmu and should be used to simulate cloning of the PCR product in pUCmu resulting a new vector called **pUCmu_geneX**.
Columns in the spreadsheet contain your values for  **geneX**, **Enz#1** and **Enz#2**.

You should also predict the final sequence of **pUCmu_geneX**. The primers should **anneal** to the template with 16-18 nucleotides.
The sequence of the **pUCmu** can be found
The size of the pUCmu can be found [here](TPs/TP03_In-silico_sub_cloning_using_ApE/pUCmu_1669_bp_cdseguid=jEg2G-c5DZWICcghJ_xmafqcsTQ.txt)

Put your answer in the **fp**, **rp** and **pUCmu\_geneX** columns. Use raw DNA sequences as indicated for Max Maximus.


© Björn Johansson 2013-2026
