# *In-silico* sub cloning

![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03.png]]

Molecular cloning often involve insertion of a gene or other DNA fragment into a plasmid (a kind of genetic vector). This new combination is called a **recombinant** DNA molecule. Cloning is necessary in order to study the function of a gene, since the cloning change the context of the gene from its original environment. Cloning is also almost always necessary in order to express the gene in a different organism, since controlling sequences such as promoters and terminators needed are different between organisms.

## Planning

It is necessary to plan a cloning experiment before doing the actual experiments. This planning should contain all the manipulations needed to reach the final construction. This exercise will show you how to make simple sequence manipulation on your computer and how to assemble the sequence of recombinant DNA molecules following a cloning strategy.

The main purpose of *in-silico* cloning is to predict the final sequence outcome of a cloning strategy or recombinant DNA construction project on the computer before performing the actual experiments in order to errors in the cloning strategy.

The final sequence is needed in order to confirm a successful cloning.

The cloning procedures in this document are done by restriction enzyme digestion followed by ligation.

You will practice cloning using a restriction enzyme with:

- 5' overhang cu
- 3' overhang cu
- blunt cu
- outside cut (type IIS enzymes)

The DNA editor ApE is used for these exercises, but any other editor could be used.

Restriction enzymes cleave DNA so that the phosphate remains on the 5' side of the cut (see below).

![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_005-1.png]]


Restriction enzymes are classified depending on the structure of the cut they make in double stranded DNA.
- If the cut in the upper (Watson) strand comes before the cut in the lower (Crick) strand, the resulting ends have 5' overhangs (see HindIII below).
- If the cut in the Watson strand comes after the cut in the Crick strand the resulting ends have 3' overhangs (see PstI below).
- If the cuts are in the same position, the cut is blunt (see EcoRV below).

![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_004-1.png]]

# Simulating 5' overhang cloning

Open a new empty window in ApE or similar DNA editor. Copy the DNA sequence below and paste it into the ApE window.

	ggtaccATGAAATAAggtacc

This sequence represents a short gene consisting of a start codon (**ATG**), one amino acid alanine codon (**AAA**) and a stop codon (**TAA**). The `ggtacc` are recognition sites for a restriction enzyme called [*Acc65*I](http://rebase.neb.com/rebase/enz/Acc65I.html). The sequence  above represents the double stranded linear DNA molecule shown below:

	ggtaccATGAAATAAggtacc
	|||||||||||||||||||||
	ccatggTACTTTATTccatgg

Now we need to find out how the enzyme *Acc65I*I cuts the DNA; we need to know which bonds are going to be broken by the enzyme.

To do this go to `Enzymes>Enzyme Selector` in ApE and look at the  *Acc65I*I enzyme. Just put the cursor above the enzyme name without clicking as shown below:

![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_002.png]]

The **^** symbol above can be understood as a cut in each strand (see below).

    -G^G-T-A-C-C-
      | | | | | |
    -C-C-A-T-G^G-

This information is also available from the on-line [restriction enzyme database](http://rebase.neb.com/rebase/enz/Acc65I.html) (see below).

![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_005.png]]

Many [companies](https://www.thermofisher.com/order/catalog/product/ER0901) sell this enzyme.

**Question 1:** Does *Acc65*I produce blunt, 5' or 3' overhangs?

**Question 2:** Which one of the following enzymes produce a blunt cut:
- BamHI
- EcoRI
- SmaI

When we digest our molecule from the previous example with *Acc65*I, we obtain **three molecules** of double stranded DNA (see below). The fragment we want to clone is the middle part.

![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_001.png]]


In order to find the *Acc65*I, site in the sequence, go to  `Enzymes>Enzyme Selector` and click ‚ÄúHighlight‚Äù at the bottom of the page.

The resulting *Acc65*I highlighting in the main sequence window can be seen below.

![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_002-1.png]]

Note how ApE indicate the cut sites in each strand using this symbol: ![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_001-1.png]].

Open the pUC19 [sequence](<TPs/TP01_Calculating_checksums_using_seguid_calculator/plasmid_pUC19_in_raw_sequence_format.txt>). in another ApE window, but keep the first sequence open as well.

Highlight the *Acc65*I site in pUC19 (see below).

![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_003.png]]

Copy the sequence between the cuts in the Watson strand (see above and below) and paste the sequence into the pUC19, exactly where the Acc65I cuts in pUC19.

![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_004.png]]
You have now simulated a cloning experiment with a restriction enzyme (and DNA ligase) that resulted in a recombinant plasmid üôÇ.
We call this plasmid **`pUC19_Acc65I_insert`**.

![|161x225](TPs/TP03_In-silico_sub_cloning_using_ApE/TP03-17.png)

**Question 3:**

The partial checksum for **`pUC19_Acc65I_insert`** is **`cdseguid=rVzaVq`** and the size is 2701 bp. What is the complete checksum of the new plasmid? How many *Acc65*I sites can you find in the new plasmid?

![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_001-1.png.gif]]

Since the sticky ends on each side of the molecule are the same, there is another way to combine the two molecules (see below.

![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_003-1.png]]

The insert fragment in the second example above has been cloned inverted. This will happen for 50% of the recombinant molecules in a real experiment. Simulate the cloning of the molecule in reverse in pUC19 using a fresh copy of the pUC19 plasmid sequence.
You can obtain the second sequence above by using the reverse complement button: ![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_014.png]] on the insert or plasmid sequence. We call the resulting plasmid **`pUC19_Acc65I_insert_b`**.


**Question 4:**  The `pUC19_Acc65I_insert_b` has 2701 bp and the partial seguid is `cdseguid=rVzaVq`. What is the complete seguid and how many *Acc65*I sites can you find in the recombinant plasmid?

## Simulating 3' overhang sticky end cloning of a short DNA fragmen

Some other enzymes such as *KpnI* or *SacI* cuts the DNA in a staggered manner so 3' sticky ends are produced. As you can see below, KpnI recognizes the same sequence as *Acc65*I, but the cut produces 3' overhangs.

![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_006.png]]
Copy the sequence below into a new ApE window.

	ggtaccATGCCCTAAggtacc

Reopen a fresh copy of the pUC19 sequence file. Recreate the same cloning experiment as in the *Acc65*I example, but with *Kpn*I.

![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_007.png]]

As before, we copy the insert between the cuts in the Watson strand and paste where *Kpn*I cuts in the pUC19.

![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_009.png]]

Create the sequence of the recombinant molecule in ApE as described above. The KpnI fragment can also enter in two directions as in the  example with *Acc65*I. Simulate the cloning of the KpnI fragment in reverse as well.

We call the resulting plasmids `pUC19_KpnI_insert` and `pUC19_KpnI_insert_b`.

**Question 5:** Fill in the missing information in the table below:

| Sequences             | Size (bp) | Partial seguid    | Complete seguid | Number of KpnI sites? |
| :-------------------- | :-------- | :---------------- | :-------------- | :-------------------- |
| `pUC19_KpnI_insert`   | 2701      | `cdseguid=I37jqk` | ?               | ?                     |
| `pUC19_KpnI_insert_b` | "         | `cdseguid=oNRmtX` | ?               | ?                     |

## Hybrid sites

Some restriction enzymes produce compatible ends, like *Bgl*II and *BamH*I below. These enzymes are [isocaudomers](https://en.wikipedia.org/wiki/Isocaudomer). These can ligate to each other as shown in the example below:

![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_013.png]]
Note that the hybrid site is not recognized by either enzyme. This means we can not cut it with either *BglI*I or *BamH*I.

**Question 6:**

The sequence below has recognition sites for *Bgl*II. Simulate the cloning of the sequence below in both orientations using pUC19:

	agatctATGAAATAAagatc

The resulting sequences are called `pUC19_BglII_BamHI_insert` and `pUC19_BglII_BamHI_insert_b`. Fill in the missing information below:

| Sequences                    | Size (bp) | Partial seguid    | Complete seguid | Number of BamHI sites? | Number of BglII sites? |
| :--------------------------- | :-------- | :---------------- | :-------------- | :--------------------- | :--------------------- |
| `pUC19_BglII_BamHI_insert`   | 2701      | `cdseguid=tPVJ7W` | ?               | ?                      | ?                      |
| `pUC19_BglII_BamHI_insert_b` | "         | `cdseguid=jN9osk` | ?               | ?                      | ?                      |
## Sticky end cloning using an outside cutter

Not all restriction enzymes cut inside the recognition sequence. The restriction enzyme ***Bsa*I**  cuts for example on the side, one and four nucleotides away from the recognition sequence (see below). More details about this enzyme can be found at [rebase](http://rebase.neb.com/rebase/enz/BsaI.html). This enzyme is used for the [Golden Gate](https://www.snapgene.com/guides/golden-gate-assembly) cloning protocol.

![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_010.png]]

As you can see above, the *Bsa*I site is not symmetric and the recognition site is not a palindrome as in the previous examples.

Reopen the pUC19 sequence file in a new window. Enter the sequence below in a new ApE window:

	GGTCTCAgatcATGAAATAAgatcAGAGACC

This molecule is cut by BsaI in the figure below:
![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_011.png]]

The overhang sequence is **NOT** defined by the recognition sequence, the actual overhang sequence depends on the sequence next to the site.
The sequence in this example was designed to be cloned in a restriction site produced by *Bam*HI for example.

Simulate the cloning of this fragment in the pUC19 vector. Create versions with both directions of the insert..

**Question 7:**
Fill in the missing information below:

| Sequences           | Size (bp) | Partial seguid    | Complete seguid | Number of BamHI sites? |
| :------------------ | :-------- | :---------------- | :-------------- | :--------------------- |
| pUC19_BsaI_insert   | 2699      | `cdseguid=BUFuz1` | ?               | ?                      |
| pUC19_BsaI_insert_b | "         | `cdseguid=k-jmfm` | ?               | ?                      |


**Question 8**:
Give an example of a cloning strategy error that could affect these experiments.


**Question 9:**

Your task is to simulate the cloning of the 1.5 kb *Bgl*II ‚Äì *EcoR*V fragment from the vector pUG6 into the *BamH*I ‚Äì *Sma*I sites of pUC19.
Get pUG6 [[vector_pUG6_in_raw_sequence_format_cdseguid=EwIUwovRBrfIP5Ga96Z36yQFfuk.txt|here]] . It is also available from GenBank under the accession number [AF298793](http://www.ncbi.nlm.nih.gov/nuccore/AF298793).

This is a directional cloning since the enzymes are not all compatible, see details for how they cut using the same technique as in **Question 3.**
This also means that there is only one result and not two as in the previous examples. This strategy is called *directional cloning* for this reason.

The size of the recombinant vector should be 4184 bp and the partial checksum is `cdseguid=THGA3G`.

What is the complete checksum?

**Question 10:**

This is an individual exercise for each student. The data you need to solve
this exercise can be found in the Google spreadsheet for this exercise where
you can find your name in the leftmost column.

One column called **sequenceZ** contains your DNA sequence that represents a
double stranded **linear** DNA molecule.

The columns **re1** and **re2** contain two restriction enzymes that cut the **sequenceZ.**
The restriction enzymes also cut the plasmid pUCmu. The sequence of
the **pUCmu** can be found [[pUCmu_1669_bp_cdseguid=jEg2G-c5DZWICcghJ_xmafqcsTQ.txt|here]].

Your task is to simulate the cloning of **sequenceZ** in **pUCmu** after digestion of
both with the two enzymes The resulting sequence where **sequenceZ** is cloned
into **pUCmu** is called **pUCmu_Z.** Put your sequence in the indicated cell.

![[TPs/TP03_In-silico_sub_cloning_using_ApE/TP03_015.png]]

Please answer with raw DNA sequences as indicated for the first example student "*Max Maximus*‚Äù. There is a 3 min YouTube video (very bad sound) here that shows how to solve this question [here](https://www.youtube.com/watch?v=ngX76ve473w&list=PLUIJEy5gZN_yt6VnhH487DNQh8VSsVO_F).


¬© Bj√∂rn Johansson 2013-2026
