# **Assembly of a tailed PCR product**
Gene deletion, or gene knockout, is one of the main ways in which the function of genes are discovered (reverse genetics).

The **yeast deletion project,** or the **Saccharomyces Genome Deletion Project**, was a global project to create a collection of gene-deletion mutants in *Saccharomyces cerevisiae*. This collection allows researchers to determine what each gene does by comparing the behavior of a mutated yeast to that of a normal yeast.

The deletion mutations made are now sold by [EUROSCARF](http://euroscarf.de/) (*European Saccharomyces Cerevisiae Archive For Functional Analysis*). The strategy used to delete the genes is described by [Kelly et al. 2001](https://pubmed.ncbi.nlm.nih.gov/18628917/). This strategy is outlined in Fig 1.

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.002.png)

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.001.png)

The strategy used has three steps (Fig 1). A PCR amplification is carried out using the UPTAG and DOWNTAG primers using a plasmid called **pFA6a-kanMX4** as template (Fig 2). The plasmid contains the KanMX4 marker that give the cell resistance to the antibiotic Geneticin.

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.004.png)

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.003.png)




In a second step, **the PCR product from the previous** step is used as template in a new PCR amplification using the UP\_45 and DOWN\_45 primers. This step was added to prolong the flanking  homologous sequences os the cassette.

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.006.png)

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.005.png)



The final PCR product contains the KanMX4 resistance gene flanked by short stretches of DNA that are identical to sequences up and downstream of the target gene. We call this PCR product a cassette.

Finally, In the third step, the cassette integrates by homologous recombination in the correct locus of the genome (Fig 4).

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.008.png)

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.007.png)

The purpose of this exercise is to assemble the sequence of a tailed PCR product from the primer sequences and the template sequence.

**Example**

We will assemble one of the KanMX4 deletion cassettes used in the *Saccharomyces* genome deletion project. This particular cassette was used to delete the CYC1 gene for cytochrome C.

**Primer sequences:**

The primer sequences in FASTA format for the CYC1 gene can be found in an **Excel sheet** called **“Yeast\_Genome\_Deletion\_Primers\_and\_Product\_Sizes.xlsx”** or in a Google [spreadsheet](https://docs.google.com/spreadsheets/d/1pVlMkCKb6iX-w-2Q3OP2iP0UMH72PEHf9TuGM4SdRZk/edit?usp=sharing). We add the gene name “CYC1” to the upper left cell (A1 in Fig 5).

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.010.png)

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.009.png)




















We copy the UPTAG, DOWNTAG, UP\_45 and DOWN\_45 primers to a text editor (Fig 6).



![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.011.png)

**Template sequence:**

The template for the first PCR with the UPTAG and DOWNTAG primers was a plasmid called pFA6a-kanMX4. You can find this plasmid in the same folder as this document in file “**pFA6a-kanMX4.txt**”. It is also available from Genbank with accession number [AJ002680](https://www.ncbi.nlm.nih.gov/nuccore/AJ002680).

The next step is to find out where the primers bind to the plasmid in order to determine the sequence of the first PCR product. There easiest way to do this is using [WebPCR](http://pydna.pythonanywhere.com/).

The expected size of this PCR product is 1617 bp. The SEGUID for this sequence is 5BF0ur17oZHgFcEctHKAer9YWkg. This sequence is available in the file **first\_CYC1\_deletion\_cassette.txt** in the folder of this file.

The second PCR product is assembled in the same way, but with the UP\_45 and DOWN\_45 primer sequences (Fig 6) and the **previously assembled cassette as template**. The second PCR product is 1671bp long and has the SEGUID **ZdMoT7KL0\_EgUFC01DddjNNfI\_E** This sequence is available in the file **second\_CYC1\_deletion\_cassette.txt** in the folder of this file.

###
### **Integration of the cassette**
The last DNA sequence we need is the genomic sequence of the *S. cerevisiae* genome where the cassette will be integrated. We can find this sequence at [SGD](https://www.yeastgenome.org/locus/S000003809/sequence), we need the sequence with +/- 1kb (Fig 7).

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.013.png)
![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.012.png)










The sequence was downloaded using the button further down on the page called “ Download Sequence (.fsa)”. This sequence is available in file **S288C\_YJR048W\_CYC1\_flanking.fsa. (uSEGUID: 3owvUzhueH77grHYK-mhR-yqkhU)**


The next step is to find out where the cassette with integrate. We open both the second cassette sequence and the genomic sequence in two windows (Fig 8).

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.015.png)
![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.014.png)

Select Tools>Align Sequences and set the options al indicated in Fig 9.

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.017.png)

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.016.png)

The alignment window show two 45-46 bp stretches of identical sequences between the cassette and the gnomic sequence.



![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.019.png)

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.018.png)

Then we copy the indicated sequences and add them as features (Select some text, then Features>New Feature>OK, Fig 11). We do this to simplify and lower the chance of mistakes.

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.021.png)

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.020.png)

Finally, the middle sequence (Fig 11, left) is replaced by the cassette ((Fig 11, right). The result can be seen in Fig 12.

![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.023.png)
![](Aspose.Words.922ad967-f9f3-4d5e-8d2c-370618f15e34.022.png)

The resulting sequence has 3632 bp and will contain 1000 bp of chromosomal DNA, the cassette and again 1000 bp chomosomal DNA. This sequence can be found in the file “**CYC1\_deletion\_locus.gb**” in the folder of this file.

**Question 1:**

This is an individual question for each student. Follow this [link](https://docs.google.com/spreadsheets/d/1Sf2jml-r693RH8V6JNE6ZCXhms6raJlOPdwEUZ_XbiU) that points to a Google Spreadsheet.  You should find your name in the leftmost column.

The column called **Standard gene name** contain the name of a gene. Your task is to assemble the first and second PCR products and the deletion locus as shown in the example.

Note that you have to get your own genomic sequence in the final step where the cassette is integrated. Use the search function at [SGD](https://www.yeastgenome.org/) to find the sequence for your gene as in Fig 7.

Put the sequences in the indicated columns:

- First PCR product
- Second PCR product
- Deletion\_locus.

**Please answer with the raw sequences as indicated for the first example student "*Max Maximus*"**. If your name is **\*not\*** in the list, please inform your instructor.
