# Golden Gate cloning

Golden Gate is a popular method for joining multiple DNA fragments together. This method is probably one of the less expensive ones, since only short DNA oligonucleotides are needed comparing to methods like Gibson assembly or homologous recombination. It is also a practical way to make combinatorial assemblies of genes. Another advantage is that the final construct does not have the recognition site of the enzyme, so it can be used again to make more modifications if needed.

Golden gate cloning is based on the special properties of type IIs restriction endonucleases. These enzymes cut on one side of the recognition sequence.

![[TPs/TP25_GoldenGateCloning/TP25_GoldenGateCloning_html_7bc530ec.png]]
_Figure 1: Restriction endonuclease II vs IIs_

The restriction enzyme BsaI (below) is often used in Golden Gate cloning. The cut site differs with the orientation as for all IIs enzymes. Note that the recognition sequence is not a [palindrome](https://en.wikipedia.org/wiki/Palindrome).

![[TPs/TP25_GoldenGateCloning/TP25_001.png]]

A simple Golden Gate strategy is depicted below. A plasmid (Destination vector) contains two cut sites for BsaI. The recognition sites are present in the short orange part.

The insert molecule (blue) also has two cut sites. The brown and green sequences are the ones that produce the sticky-ends. I
In the vector, the actual . In the insert, the recognition sites are present in the gray flanking sequences.

Take careful note of the direction of each site.

![[TPs/TP25_GoldenGateCloning/TP25_002.png]]

The brown and the light green sequences are the cut sites (below). Select and highlight the BsaI sites in both sequences.

![[TPs/TP25_GoldenGateCloning/TP25_003.png]]

Copy/paste the appropriate sequence from the insert to the Destination vector (above). The size of the resulting vector is 1558 bp.
The DNA sequences of the two molecules shown below are available in the two files [Destination_vector.gb](<TPs/TP25_GoldenGateCloning/Destination_vector.gb>) and [insert.gb](<TPs/TP25_GoldenGateCloning/insert.gb>).

Look at the figure below for a detailed overview of the process.

![](TPs/TP25_GoldenGateCloning/TP25_GoldenGateCloning_html_dc55f6fe.png)

## Question 1:

The partial seguid checksum of the previous example is`cdseguid=xZpuKw`. What is the complete checksum?

## Question 2:

Since restriction cut sites are almost never present where we need them, we usually have to introduce them by PCR. This is done by adding the desired nucleotides to the 5’ end of primers for the target insert, much in the same way as for normal type II restriction enzymes (see TP04).

The only difference is that you have to define the sticky-ends yourself. Go to the Google Spreadsheet for this question. Find your name in the leftmost column.

The column called `geneZ` contains a DNA sequence that represents a double stranded linear DNA molecule that is also an open reading frame for a gene.

Your task is to design two primers (**fp** and **rp**) for  geneZ  that will amplify the entire sequence. The primers should anneal with the template with a length of 18-22 bp. This means that in this case, the total primer will be 6 + 1 + 4 + 18-22 = 29-33

![[TPs/TP25_GoldenGateCloning/TP25_005.png]]

Each primer should contain one recognition site for the BsaI restriction enzyme. The restriction sites should produce sticky-ends compatible with the **pGG** golden gate vector. The pGG vector is different for each student and can be found in the spreadsheet.

Put your results in the indicated cells for forward primer (**fp**), reverse primer (**rp**). Put the sequence of the resulting PCR product in the **PCR product** cell. Put the resulting vector in the **pGG_geneZ** column. Please answer with raw DNA sequences as indicated for the first example student "Max Maximus".
## Optional - videos:

These two YouTube videos has a nice introduction to Golden Gate cloning.

![[TPs/TP25_GoldenGateCloning/TP25_GoldenGateCloning_html_3ca116a9.png]]
[above (2m17s)](https://youtu.be/EpHeu44hitI?si=IBkYdH1M_uMLdQ68)

![[TPs/TP25_GoldenGateCloning/TP25_GoldenGateCloning_html_4e5dac23.png]]

[above (3 min)](https://youtu.be/FTkSY3evOoY?si=5abAXOY9pRpOfzrb)


## Optional - Using the Golden Gate Assembler in ApE

ApE has a Golden Gate assembler that can both design and assemble automatically. If you open at least two ApE windows simultaneously, you can select Tools>Golden Gate Assembler (Figure 9). Using this tool is optional.

![[TPs/TP25_GoldenGateCloning/TP25_GoldenGateCloning_html_b76887ff.png]]
![[TP25_GoldenGateCloning_html_4f68014d.gif]]



Select the BsaI restriction enzyme and click “ok” (Figure 10).

![[TPs/TP25_GoldenGateCloning/TP25_GoldenGateCloning_html_fb5ec163.png]]




![[TPs/TP25_GoldenGateCloning/TP25_GoldenGateCloning_html_e0ff26d5.png]]
